tasks/repositories.yml

```
@@ -20,7 +20,12 @@
     - "deb http://archive.ubuntu.com/ubuntu {{ ansible_distribution_release | lower }}-updates multiverse"
     - "deb-src http://archive.ubuntu.com/ubuntu {{ ansible_distribution_release | lower }}-updates multiverse"
     - "deb http://repo.mongodb.org/apt/ubuntu/ {{ ansible_distribution_release | lower }}/mongodb-org/{{ bbb_mongodb_version }} multiverse"
-    - "deb {{ bbb_apt_mirror }}/bionic-230/ bigbluebutton-bionic main"
+
```

Hi @BrutalBirdie,
you should add a cleanup-task for the old sources-list file:
`/etc/apt/sources.list.d/ubuntu_bigbluebutton_org_bionic_230.list`.
```suggestion

- name: remove deprecated bigbluebutton sources-list file
  file:
    path: '/etc/apt/sources.list.d/ubuntu_bigbluebutton_org_bionic_230.list'
    state: absent

```
Btw. which sources-list file would bbb-install.sh create?